.github/workflows/build.yml

```diff
@@ -85,11 +85,17 @@ jobs:
       with:
         submodules: true
 
-    - name: Set up Java 11
+    - name: Set up Java 17
       uses: actions/setup-java@v2
       with:
         distribution: 'adopt'
-        java-version: '11'
+        java-version: '17'
+
+    - name: Enable KVM
+      run: |
+        echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+        sudo udevadm control --reload-rules
+        sudo udevadm trigger --name-match=kvm
```

Comment:
Yikes! Yes, we need the `Enable KVM` step because we [switched to Linux](https://github.blog/changelog/2023-02-23-hardware-accelerated-android-virtualization-on-actions-windows-and-linux-larger-hosted-runners/) but these changes here are unnecessary since they just duplicate the `Enable KVM` step we already have below.  I think this was just a merge error on my part.  Removing.